### PR TITLE
Show hidden files + Update locate database

### PR DESCRIPTION
**Pull request by selfagency**
Revised as requested by @bpinto

### CODE REVIEW

File: showhidden.fish [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function showhidden -d "Hides/reveals system files and folders in Finder"
+  if test "yes" = (echo $argv | tr '[:upper:]' '[:lower:]')
```
Comment (bpinto) [inline showhidden.fish]: A `switch` statement might be simpler:

```
switch "$argv"
case Yes YES yes
    ...
case No NO no
    ...
case '*'
    echo "Command input must be 'yes' or 'no'"
end
```

Ref: http://fishshell.com/docs/current/tutorial.html#tut_conditionals

**Comment by bpinto** [comment]
:+1: Looks good to me.

Ping @scorphus

File: showhidden.fish (hunk 1 above)
Comment (selfagency) [inline showhidden.fish]: Done

**Comment by selfagency** [comment]
what up @scorphus

**Comment by bpinto** [comment]
Hey @selfagency could you rebase it?

**Comment by selfagency** [comment]
i'm not sure i understand -- if i rebase it, doesn't mean that i'm pulling the head to my local copy?

**Comment by bpinto** [comment]
It would reapply your commits on top of the latest commit from master.

**Comment by selfagency** [comment]
how do i do that with a forked branch that's already its own head?

**Pull request closed without merging**